src/ethereum/osaka/vm/instructions/environment.py

```diff
@@ -19,7 +19,6 @@
 from ethereum.utils.numeric import ceil32
 
```

Comment:
blockhash was not modified, but that should also likely be a part of "state" since 2935 has been implemented